tests/x11/vlc.pm

```diff
@@ -44,7 +44,7 @@ sub run {
         assert_and_click 'close_vlc';
     }
 
-    x11_start_program('vlc --no-autoscale --start-paused --start-time=19.5 data/Big_Buck_Bunny_8_seconds_bird_clip.ogv', target_match => 'vlc-playing');
+    x11_start_program('vlc --no-autoscale --loop data/test.ogv', target_match => 'vlc-playing');
     assert_screen 'vlc-playing';
```

Comment:
is that assert_screen actually still needed, with the implicit x11_start_program target_match?